### ISSUE TITLE
Warning: Accordion: isMounted is deprecated.

### ISSUE THREAD

**Issue by Zomis** [naoufal/react-native-accordion#54]
When using an accordion in React Native (using Create React Native App) I get this warning as soon as I open the accordion:

> Warning: Accordion: isMounted is deprecated. Instead, make sure to clean up subscriptions and pending requests in componentWillUnmount to prevent memory leaks.

**Comment by lyudad** [comment]
I have the same, when click on Accordion header

**Comment by juanpdelat** [comment]
Is this project still maintained?

**Comment by manojxio** [comment]
I have the same issue, when you click on the Accordion header, nothing happens just the warning.

**Comment by thatchej** [comment]
Yep same exact issue here

**Comment by Zomis** [comment]
@cipherm For me, it works to click it - The content does show up, but this warning message shows up as well.

**Comment by Crael94** [comment]
@Zomis did you fix it ? 

thanks you

**Comment by Zomis** [comment]
@Crael94 No, I still get this warning. And this issue isn't really mine to fix.
